From f3ace52de88566529e304adce0d528d12533f676 Mon Sep 17 00:00:00 2001
From: Sasha Levin <sashal@kernel.org>
Date: Mon, 11 Aug 2025 15:00:01 +0200
Subject: gpio: TODO: remove the task for converting to the new line setters

From: Geert Uytterhoeven <geert+renesas@glider.be>

[ Upstream commit 6ec4b94e8e959b4201ca0bfc43fa50dc946d10cb ]

The task is complete, but this was not reflected in the TODO file.

Fixes: d9d87d90cc0b10cd ("treewide: rename GPIO set callbacks back to their original names")
Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
Link: https://lore.kernel.org/r/417af7e5a110c527eb759289bd5d2fd6885f4e01.1754917104.git.geert+renesas@glider.be
Signed-off-by: Bartosz Golaszewski <bartosz.golaszewski@linaro.org>
Signed-off-by: Sasha Levin <sashal@kernel.org>
---
 drivers/gpio/TODO | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/drivers/gpio/TODO b/drivers/gpio/TODO
index 7a09a4f58551b..b797499e627ee 100644
--- a/drivers/gpio/TODO
+++ b/drivers/gpio/TODO
@@ -176,18 +176,6 @@ cannot be converted yet, but watch this space!
 
 -------------------------------------------------------------------------------
 
-Convert all GPIO chips to using the new, value returning line setters
-
-struct gpio_chip's set() and set_multiple() callbacks are now deprecated. They
-return void and thus do not allow drivers to indicate failure to set the line
-value back to the caller.
-
-We've now added new variants - set_rv() and set_multiple_rv() that return an
-integer. Let's convert all GPIO drivers treewide to use the new callbacks,
-remove the old ones and finally rename the new ones back to the old names.
-
--------------------------------------------------------------------------------
-
 Remove legacy sysfs features
 
 We have two parallel per-chip class devices and per-exported-line attribute
-- 
2.51.0

